lua/smart-splits/config.lua

```diff
@@ -128,6 +127,14 @@ function M.setup(new_config)
     mux_utils.startup()
   end
 
+  -- check for incompatible settings
+  if mux_utils.are_we_kitty() and config.at_edge == 'wrap' then
```

Comment:
We should check the configured mux here, it works fine in Kitty if you're using a different mux like tmux or zellij right?

Reply:
Something like this? 
`if mux_utils.are_we_kitty() and config.at_edge == 'wrap' and multiplexer_integration == nil then`